app/controllers/catalog_controller.rb

```diff
@@ -8,7 +8,7 @@ class CatalogController < ApplicationController
 
   # POST /catalog
   def create
-    druid = poh_params[:druid]
+    druid = poh_params[:druid].split(':').last
```

Comment:
Should go ahead and use `split(':', 2)` since we only ever want to split off at most 1 chunk.

Reply:
when i do this, it breaks all my tests

Reply:
oh nvm. i got it